README.md

```diff
@@ -1,39 +1,37 @@
-# tribune-client
-Tribune TTS gRPC client
+# tts-client
+Techmo TTS gRPC client
```

Comment:
Techmo TTS gRPC client ->

This repository contains Techmo TTS gRPC client implementations.

Reply:
Changed to `This repository contains Techmo TTS gRPC service API and client implementations.`